Userland/Libraries/LibCore/System.cpp

```diff
@@ -368,13 +368,30 @@ ErrorOr<void> fchmod(int fd, mode_t mode)
     return {};
 }
 
+ErrorOr<void> lchown(StringView pathname, uid_t uid, gid_t gid)
+{
+    if (!pathname.characters_without_null_termination())
+        return Error::from_syscall("chown"sv, -EFAULT);
+
+#ifdef __serenity__
+    Syscall::SC_chown_params params = { { pathname.characters_without_null_termination(), pathname.length() }, uid, gid, AT_FDCWD, false };
+    int rc = syscall(SC_chown, &params);
+    HANDLE_SYSCALL_RETURN_VALUE("chown"sv, rc, {});
+#else
+    String path = pathname;
+    if (::chown(path.characters(), uid, gid) < 0)
```

Comment:
Looks like a copy paste error here, should probably call ::lchown

Reply:
And should change to "lchown" in the Error strings

Reply:
Ah, yeah

Reply:
> And should change to "lchown" in the Error strings

Where?

Reply:
I see now, will fix tommorow.

Reply:
Fixed